src/components/10-atoms/title-section/index.js

```diff
@@ -0,0 +1,44 @@
+import { LitElement, html, css, unsafeCSS } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map';
+/* eslint-disable import/no-extraneous-dependencies */
+import defineOnce from '../../../utils/define-once';
+import styles from './index.scss';
+
+class AXATitleSection extends LitElement {
+  static get tagName() {
+    return 'axa-title-section';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    return {
+      variant: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    this.variant = '';
+  }
+
+  render() {
+    const classes = {
+      'a-title-section--white': this.variant === 'white',
+    };
+
+    return html`
+      <h2 class="a-title-section ${classMap(classes)}">
```

Comment:
why h2? and if h2, where is h1?

Reply:
it should be `<h1>` and in the storybook examples title should be surrounded via a `article`

Reply:
It is a section title. Sections can exist mulitple times on a page, so it is mostly an H2 or H3.

H1 is a page title. Our story only previous the component itself. As a consumer you will have your h1 "somewhere"

Reply:
Per HTML5 you can have a h1 per self contained block element like article and section therefore please use h1, even if user has it somewhere on the page

Reply:
You are referring to HTML4, which we don’t support

Reply:
https://html.spec.whatwg.org/multipage/sections.html#headings-and-sections

Reply:
Quote: “Authors are also encouraged to explicitly wrap sections in elements of sectioning content, instead of relying on the implicit sections generated by having multiple headings in one element of sectioning content.”

Sectioning content: https://html.spec.whatwg.org/multipage/dom.html#sectioning-content-2